iib/web/api_v1.py

```diff
@@ -37,3 +38,94 @@ def add_bundles():
 
     flask.current_app.logger.debug('Successfully scheduled request %d', request.id)
     return flask.jsonify(request.to_json()), 201
+
+
+@api_v1.route('/requests/<int:request_id>', methods=['PATCH'])
```

Comment:
```suggestion
@api_v1.route('/builds/<int:request_id>', methods=['PATCH'])
```

Reply:
I thought we decided on keeping it `requests` since it's an internal endpoint used to patch requests.

Reply:
To be consistent and "RESTful", we should use the same name.